ext/crypto/shared.rs

```diff
@@ -144,7 +144,7 @@ impl RawKeyData {
             )
           })?;
         let point = p384::EncodedPoint::from_bytes(&ec_key.encoded_point)
-          .map_err(|_| data_error("expected valid public EC key"))?;
+          .map_err(|_| data_error("expected valid private EC key"))?;
```

Comment:
This is the public encoded point - the error should not be for private key decoding.

Reply:
Oops, fixed.